src/Diagnostics.DataProviders/KustoHeartBeatService/KustoHeartBeatService.cs

```diff
@@ -50,9 +51,10 @@ public void Dispose()
             _cancellationToken.Cancel();
         }
 
-        public KustoHeartBeatService(KustoDataProviderConfiguration configuration)
+        public KustoHeartBeatService(KustoDataProviderConfiguration configuration, IHostingEnvironment hostingEnvironment)
         {
             _configuration = configuration;
+            runHeartBeatQuery = hostingEnvironment.IsProduction() || hostingEnvironment.IsEnvironment("UnitTest");
```

Comment:
should we control this via settings Rekha?
It will give us flexibility to turn it on/off on mulitple environments in future.

Reply:
Okay sure